README.md

```diff
@@ -1,13 +1,46 @@
-# Project Todos
+# Project Todos ✏️
 
-Replace this readme with your own information about your project.
+A todo-app built with React and Redux. The app lists all todo tasks - completed or uncompleted. The state of each todo can be shown as either done or not done with checking a checkbox. New todos are added using a text field. Todos can be removed one by one or all at once using the clear-all button. After pressing clear-all, a warning alert box appears asking if user wants to clear all or cancel that action. The app also counts and shows how many of the listed todos are done.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+## View it live 👀
 
-## The problem
+https://app.netlify.com/sites/quirky-kepler-b954a8
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+## Learnings 🤓
+
+- The role redux-toolkit plays
+- How to create new slices
+- How to create new reducers
+- How to create new actions
+- What immutability means and how to work with it
+
+- How to structure a redux store
+- How to write reducers and actions
+- How to interact with your store using selectors or by dispatching actions
+
+## More features
```

Comment:
🎉 Super impressive to see you have added all these extra features! And also that you took the time to do accessibility testing 👌